api/model.go

```diff
@@ -60,6 +60,8 @@ type Volume struct {
 	DataEngine                       longhorn.DataEngineType                `json:"dataEngine"`
 	OfflineReplicaRebuilding         longhorn.OfflineReplicaRebuilding      `json:"offlineReplicaRebuilding"`
 	OfflineReplicaRebuildingRequired bool                                   `json:"offlineReplicaRebuildingRequired"`
+	SnapshotMaxCount                 int                                    `json:"snapshotMaxCount"`
+	SnapshotMaxSize                  string                                 `json:"snapshotMaxSize"`
```

Comment:
Why the type is string?

Reply:
It's like `Size`. We use string for it.

https://github.com/longhorn/longhorn-manager/blob/591946024a333421dc446afeea62618525fdb42a/api/model.go#L33

Reply:
I see. Should revisit why we did that way later.